src/components/common/SVG/Components/SearchSVG.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const SearchSVG = () => {
+  return (
+    <svg
+      className={SearchSVG}
+      width="24px"
+      height="24px"
+      xmlns="http://www.w3.org/1999/xhtml"
+      viewBox="0 0 24 24"
+      role="img"
+    >
+      <g fill="#fdd">
```

Comment:
Please remove the fill to allow customization on usage.
```suggestion
      <g>
```